src/tests/api.test.ts

```diff
@@ -219,6 +219,10 @@ describe('Api members', () => {
   // });
   // console.log('res', res);
   // });
+<<<<<<< HEAD
+
+=======
+>>>>>>> 9697799310f22736934e1ef30254cce1c7c00fe0
```

Comment:
there are merge artifacts in multiple places - that should be all cleaned up an resubmitted